src/dotnet/commands/dotnet-build/Program.cs

```diff
@@ -15,13 +15,19 @@ namespace Microsoft.DotNet.Tools.Build
 {
     public class BuildCommand
     {
-        public static int Run(string[] args)
+        public static int Run(string[] args) => Run(args, null);
+
+        public static int Run(string[] args, WorkspaceContext workspace)
```

Comment:
This change means that in-box commands have a distinct performance advantage over extensions. We need to get to a model where this WorkspaceContext can be accessed from across process boundaries. How will we make this perf benefit available to other commands?